### ISSUE TITLE
[Question] Persist Timer Tasks

### ISSUE THREAD

**Issue by dev-andremonteiro** [my8100/scrapydweb#243]
**Is there any way to persist timer tasks between restarts ?**

Currently using this library in my docker image and I wonder if there is any way to persist the timer tasks that are running between restarts. I found out about how to persist jobs with `scrapyd.jobstorage.SqliteJobStorage` but I still need to persist the timer tasks. 

An alternative would be to have some kind of internal API for me to hit when my docker instance starts to restart my timer tasks, but having a way to persist them would be better.

Thank you for all your work! :)

**Comment by my8100** [comment]
Please refer to the DATABASE_URL parameter.
https://github.com/my8100/scrapydweb/blob/19d807359e8833dbde1ccc1352700dfaf7fbf579/scrapydweb/default_settings.py#L368

**Comment by dev-andremonteiro** [comment]
Confirmed that using DATABASE_URL works for storing timer tasks between restarts. 
**Solved**  ✅


Now, just a quick note: 

I had to change the code here

```scrapydweb/utils/setup_database.py
    conn = psycopg2.connect(host=host, port=int(port), user=username, password=password)
```

to this

```scrapydweb/utils/setup_database.py
    conn = psycopg2.connect(dbname='postgres', host=host, port=int(port), user=username, password=password)
```

and make sure I am using this exact format for DATABASE_URL: `postgresql://username:password@localhost:5432`
even if you don't have a password setup. If you don't use that format and have a blank password it looks like the username filled up is `username:` and the code crashes.
